k8s/production/custom/prune-buildcache/cron-jobs.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: prune-buildcache
+  namespace: custom
+spec:
+  schedule: "0 0 * * 6"
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      activeDeadlineSeconds: 1200 # terminate any running job after 20 minutes
+      backoffLimit: 0
+      template:
+        spec:
+          serviceAccountName: prune-buildcache
+          restartPolicy: Never
+          containers:
+          - name: pruner
+            image: ghcr.io/spack/ci-prune-buildcache:0.0.1
```

Comment:
You'll also need to add an entry to the job matrix in this workflow https://github.com/spack/spack-infrastructure/blob/main/.github/workflows/custom_docker_builds.yml to build this image.

Reply:
Fixed in https://github.com/spack/spack-infrastructure/pull/415/commits/c9aee27233c62764e79b7f738884bed5a64742d4